docs/modules/ROOT/pages/appendices.adoc

```diff
@@ -0,0 +1,27 @@
+= Appendices
```

Comment:
This page is quite big, would it make sense to have each appendix as a nested document just like in the core and advanced features ?

Reply:
making sense. I did the same that spring-pulsar is doing.
However, we loose automatic appendix labels (A.1, A.2, etc ...).
Actually, to be able to generate appendix labels, we need to include all appendices in one single page ...


anyway, I tend to think that it is now better to have shorter separate pages for the appendices, and spring-pulsar is doing the same: https://docs.spring.io/spring-pulsar/docs/current/reference/appendix/version-compatibility.html

Fixed in https://github.com/reactor/reactor-core/pull/3806/commits/63d66615d18724a709dbc6841b763dbf22fd71b9

Reply:
I think it works much much better now, thanks. I don't think the labeling is as important as the content and ease of access.